exercises/simple-linked-list/linked_list.exs

```diff
@@ -0,0 +1,83 @@
+defmodule LinkedList do
+  @opaque t :: tuple()
+
+  @doc """
+  Construct a new LinkedList
+  """
+  @spec new() :: t
+  def new() do
+    # Your implementation here...
```

Comment:
Yeah, we remove the comments and leave the function that we expect them to implement just empty (along with the spec).

I mostly agree with you in general that it would be preferable to not give the user anything so they have to "completely" TDD their solution. I'm sure that somewhere before that discussion has been had and people came up with really good reasons for doing it this way, so I just roll with it for now 😉 

One reason that I think is really good - especially in Elixir - is that we can give them the typespec for each function that we expect them to implement when we give them the file. I think that alone makes it all worth it. I can see how getting a user to implement those on their own would be good, but I can't think of tests that would drive them to write those typespecs. Is there a way to test that a typespec is implemented for a function?